src/libs/actions/App.js

```diff
@@ -185,6 +185,11 @@ function getOnyxDataForOpenOrReconnect(isOpenApp = false) {
         ],
     };
     if (!isOpenApp) {
+        defaultData.optimisticData.push({
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.NVP_IS_FIRST_TIME_NEW_EXPENSIFY_USER,
+            value: false,
+        });
```

Comment:
Looking for a better solution to this. What happens is that without this we end up waiting around forever for the `serverDataIsReadyPromise()` to finish because this value never gets set.